build/images/scripts/start_ovs_windows_containerd.ps1

```diff
@@ -0,0 +1,171 @@
+$ErrorActionPreference = "Stop"
+$OVSVersion = "3.0.5-antrea.0"
+$OVSDownloadURL = "https://downloads.antrea.io/ovs/ovs-$OVSVersion-win64.zip"
+$OVSPublishedHash = 'fd27703ef7314b26b98cffb7aea27d569530ebd3ac3c98daa981ca2654373032'
+$WorkDir = [System.IO.Path]::GetDirectoryName($myInvocation.MyCommand.Definition)
+$OVSDownloadDir = $WorkDir
+$OVSInstallDir = "C:\openvswitch"
+$OVSZip = "$OVSDownloadDir\ovs-win64.zip"
+$ImportCertificate = $true
+$CheckFileHash = $true
+$PowerShellModuleBase = "C:\Windows\System32\WindowsPowerShell\v1.0\Modules"
+$InstallLog = "$OVSDownloadDir\install_ovs.log"
+
+function WaitExpandFiles($Src, $Dest) {
+    Write-Host "Extract $Src to $Dest"
+    Expand-Archive -Path $Src -DestinationPath $Dest | Out-Null
+}
+
+function CheckIfOVSInstalled() {
+    if (Test-Path -Path $OVSInstallDir) {
+        Write-Host "$OVSInstallDir already exists, OVS is already installed."
+        return $true
+    }
+    else {
+        Write-Host "$OVSInstallDir does not exist, OVS is not installed."
+        return $false
+    }
+}
+
+function CreatePath($Path){
+    if ($(Test-Path $Path)) {
+        mv $Path $($Path + "_bak")
+    }
+    mkdir -p $Path | Out-Null
+}
+
+function DownloadOVS() {
+    If (!(Test-Path $OVSDownloadDir)) {
+        mkdir -p $OVSDownloadDir
+    }
+    Write-Host "Downloading OVS package from $OVSDownloadURL to $OVSZip"
+    curl.exe -sLo $OVSZip $OVSDownloadURL
+    If (!$?) {
+        Write-Host "Download OVS failed, URL: $OVSDownloadURL"
+        exit 1
+    }
+    if ($CheckFileHash) {
+        $FileHash = Get-FileHash $OVSZip
+        If ($OVSPublishedHash -ne "" -And $FileHash.Hash -ne $OVSPublishedHash) {
+            Write-Host "SHA256 mismatch for OVS download"
+            exit 1
+        }
+    }
+
+    Write-Host "Download OVS package success."
+}
+
+function InstallOVS() {
```

Comment:
It looks you plan to use start_ovs_windows_containerd.ps1 to replace Run-Antrea-OVS-Container in the antrea-ovs container in antrea-agent Pod. But why do you introduce so many intallation related steps in the script which really requires ovsdb-server and ovs-vswitchd running.

I think this PR is to support installing OVS driver in hostProcess Pod (not on the Windows host), then we shall place the logic in "init-containers" stage if necessary rather than the antrea-ovs container, right? As for they other copy operations, it is not required.

Reply:
After new changes , the netcfg command is being used to install the driver before running the ovs processes. Also , we do not need to have the openvswitch directory on host .